components/outgrow/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@pipedream/outgrow",
+  "version": "0.0.1",
+  "description": "Pipedream Outgrow Components",
+  "main": "outgrow.app.mjs",
+  "keywords": [
+    "pipedream",
+    "outgrow"
+  ],
+  "homepage": "https://pipedream.com/apps/outgrow",
+  "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
+  "license": "MIT",
+  "gitHead": "e12480b94cc03bed4808ebc6b13e7fdb3a1ba535",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@pipedream/platform": "^0.10.0"
+  }
+}
```

Comment:
```suggestion
}

```

Reply:
I didn't really get this suggestion. The file already has a new line at the end of the file 🤔

Reply:
Weird, but resolved so!